### PR TITLE
docs: fix incomplete member of address in cheatsheet.rst

### PR DESCRIPTION
**Pull request by hafeidejiangyou**
As mentioned, there are some missing contents in the table. It will complete them here.

### CODE REVIEW

**Review by matheusaaguiar — CHANGES_REQUESTED** [review]
Hi @hafeidejiangyou , thanks for your contribution. 
Some small corrections and suggestions:
 - The entries should be sorted by alphabetical order. 
 - The description of the newly added members should follow the pattern of the previous ones, explicitly referring to `address`. For example, `issue low-level CALL of function implemented by address ...`

File: docs/cheatsheet.rst [hunk 1]
```diff
@@ -46,6 +46,12 @@ Members of ``address``
 - ``<address>.balance`` (``uint256``): balance of the :ref:`address` in Wei
 - ``<address>.code`` (``bytes memory``): code at the :ref:`address` (can be empty)
 - ``<address>.codehash`` (``bytes32``): the codehash of the :ref:`address`
+- ``<address>.call(bytes memory) returns (bool, bytes memory)``: issue low-level ``CALL`` with the given payload implemented by :ref:`address`,
```
Comment (matheusaaguiar) [inline docs/cheatsheet.rst]: This entry should be before `<address>.code`.

Comment (hafeidejiangyou) [reply docs/cheatsheet.rst]: Do you think it's good to arrange things in alphabetical order like this? Can't methods and properties be put together? By the way, the other parts are not completely in alphabetical order, why bother with this on a document that is overall very poor?

Comment (matheusaaguiar) [reply docs/cheatsheet.rst]: Yeah, I didn't think about maintaining the methods and properties grouped when I suggested that. You have a good point. Let's keep it as is.



> why bother with this on a document that is overall very poor?

Hopefully we can improve that, bit by bit, with the help of the community :)

**Review by matheusaaguiar — APPROVED** [review]

**Comment by r0qs** [comment]
I tried to rebase your branch and do a minor adjustment so we could merge it, but you created the PR from your develop branch instead of a feature branch and I couldn't properly rebase it. I'm sorry. But I'm opening another PR cherry-picking your commit.

**Pull request closed without merging**